src/main/java/nextstep/optional/Expression.java

```diff
@@ -14,12 +16,10 @@ private static boolean matchExpression(Expression e, String expression) {
     }
 
     static Expression of(String expression) {
-        for (Expression v : values()) {
-            if (matchExpression(v, expression)) {
-                return v;
-            }
-        }
+        return Arrays.stream(values())
+                .filter(v->Expression.matchExpression(v,expression))
```

Comment:
클래스 내부 `method` 호출이니 `Expression`는 없어도 될 것 같습니다 🙏 
```suggestion
                .filter(v -> matchExpression(v, expression))
```